content/en/docs/security/_index.md

```diff
@@ -41,12 +41,18 @@ These include:
     - References `cluster-admin` `ClusterRole`
     - Bound to service accounts for only `kustomize-controller` and `helm-controller`
 
-Flux uses these two `ClusterRoleBinding` strategies in order to follow the principle of least privilege, and to allow for clear access separation using tools purpose built for policy enforcement (OPA, Kyverno, admission controllers).
+Flux uses these two `ClusterRoleBinding` strategies in order to allow for clear access separation using tools purpose built for policy enforcement (OPA, Kyverno, admission controllers).
 
 For example, the design allows all controllers to access Flux CRDs (binds to `crd-controller` `ClusterRole`), but only binds the Flux reconciler controllers for Kustomize and Helm to `cluster-admin` `ClusterRole`, as these are the only two controllers that manage resources in the cluster.
 
-However in a soft multi-tenancy setup, Flux does not reconcile a tenant's repo under the `cluster-admin` role.
-Instead you specify a different service account in your manifest, and the Flux controllers will use the Kubernetes Impersonation API under `cluster-admin` to impersonate that service account.
+However in a [soft multi-tenancy setup]({{< relref "../get-started#multi-cluster-setup" >}}), Flux does not reconcile a tenant's repo under the `cluster-admin` role.
```

Comment:
:+1: (checked that the fancy markup turns into something sensible, in the preview.)